pkg/kubelet/kubelet.go

```diff
@@ -926,7 +926,10 @@ func (kl *Kubelet) initializeModules() error {
 // initializeRuntimeDependentModules will initialize internal modules that require the container runtime to be up.
 func (kl *Kubelet) initializeRuntimeDependentModules() {
 	if err := kl.cadvisor.Start(); err != nil {
-		kl.runtimeState.setInternalError(fmt.Errorf("Failed to start cAdvisor %v", err))
+		kl.runtimeState.setInternalError(fmt.Errorf("failed to start cAdvisor %v", err))
+	}
+	if err := kl.evictionManager.Start(kl, kl.getActivePods, evictionMonitoringPeriod); err != nil {
```

Comment:
Please add comment to indicate that eviction manager has to start after cadvisor.